### ISSUE TITLE
OnClick events are not propagating for desktop apps on macOS

### ISSUE THREAD

**Issue by esimkowitz** [DioxusLabs/dioxus#1068]
**Problem**

I am unable to listen for OnClick events when running a dioxus-desktop project on macOS 13.4. The onclick handlers never seem to get invoked. I am not experiencing any issues when using other handlers such as oninput for text boxes.

**Steps To Reproduce**

- Create a simple application such as the following: [dioxus-onclick-repro](https://github.com/esimkowitz/dioxus-onclick-repro) with a listener that prints every time it sees an onclick event for a button.
- `cargo run` on macOS 13.4
- Click the button and observe that no print statements are logged to the console.

**Expected behavior**

The above example should produce logs to the console every time the button is pressed.

**Environment:**
 - Dioxus version: 0.3.2
 - Dioxus-Desktop version: 0.3.0
 - Rust version: 1.70.0
 - OS info: macOS 13.4
 - App platform: desktop

**Questionnaire**

- [x] I'm interested in fixing this myself but don't know where to start
- [ ] I would like to fix and I have a solution
- [ ] I don't have time to fix this right now, but maybe later

**Comment by esimkowitz** [comment]
This appears to be breaking as of 0.3, as it works fine in 0.2

**Comment by DogeDark** [comment]
Looks like this has already been fixed when using the master branch.
As a workaround until the next Dioxus release:
```toml
[dependencies]
dioxus = { git = "https://github.com/DioxusLabs/dioxus" }
dioxus-desktop = { git = "https://github.com/DioxusLabs/dioxus"}
```

**Comment by esimkowitz** [comment]
Thanks for the heads-up, can confirm with this workaround resolves my issue! Looking forward to the next release!

**Comment by c-dem** [comment]
Just to add more data on this issue: I seem to have also experienced this problem on Windows 11. Using the latest version from Github also fixes my issue.  However there may be a related Rust version issue. I had created a new crate with Rust 1.69 a week ago using:
```
dioxus = "0.3.2"
dioxus-desktop = "0.3.0"
```
The onclick events worked fine.  Interestingly they continue to work fine after I upgraded to Rust 1.70.  However when I created a new crate today with Rust 1.70 onclick events no longer worked. I even tried using the same cargo.toml and simple main.rs file.  I ran "cargo clean" on both crates.  The crate I first created under Rust 1.69 always has working onclick events and the one I created under Rust 1.70 did not - even using the same version of Dioxus. I don't know if there is some weird Rust 1.70 caching issue, but I wanted to add my observations.
